addon/components/docs-code-highlight/index.js

```diff
@@ -5,7 +5,6 @@ import hljs from 'highlight.js/lib/core';
 import javascript from 'highlight.js/lib/languages/javascript';
 import css from 'highlight.js/lib/languages/css';
 import handlebars from 'highlight.js/lib/languages/handlebars';
-import htmlbars from 'highlight.js/lib/languages/htmlbars';
```

Comment:
`htmlbars` is no longer a thing in highlight.js, being replaced with just `handlebars`.